backend/custom_models/migrations/0013_upload_background_tree_view_info.py

```diff
@@ -0,0 +1,636 @@
+                language=l_english,
+            ),
+            item_text(
+                item_id=i_tree_view_of_team_info_person_3,
+                text_detail="Serge Duplessis - Analyste de l’assurance de la qualité",
+                language=l_french,
+            ),
+            item_text(
+                item_id=i_tree_view_of_team_info_person_4,
+                text_detail="Marina Richter - QA Analyst",
+                language=l_english,
+            ),
+            item_text(
+                item_id=i_tree_view_of_team_info_person_4,
+                text_detail="Marina Richter - Analyste de l’assurance de la qualité",
+                language=l_french,
+            ),
+            item_text(
+                item_id=i_tree_view_of_team_info_person_5,
+                text_detail="Mary Woodside - QA Analyst",
+                language=l_english,
+            ),
+            item_text(
+                item_id=i_tree_view_of_team_info_person_5,
+                text_detail="Mary Woodside - Analyste de l’assurance de la qualité",
+                language=l_french,
+            ),
+            item_text(
+                item_id=i_tree_view_of_team_info_person_6,
+                text_detail="Charlie Wang - QA Analyst",
+                language=l_english,
+            ),
+            item_text(
+                item_id=i_tree_view_of_team_info_person_6,
+                text_detail="Charlie Wang - Analyste de l’assurance de la qualité",
+                language=l_french,
+            ),
+            item_text(
+                item_id=i_tree_view_of_team_info_person_7,
+                text_detail="Jack Laurier - QA Analyst",
+                language=l_english,
+            ),
+            item_text(
+                item_id=i_tree_view_of_team_info_person_7,
+                text_detail="Jack Laurier - Analyste de l’assurance de la qualité",
+                language=l_french,
+            ),
+        ]
+    )
+
+
+def destroy_background_tree_view_info(apps, schema_editor):
+    # get models
+    language = apps.get_model("custom_models", "Language")
+    item_type = apps.get_model("custom_models", "ItemType")
+    item = apps.get_model("custom_models", "Item")
+    item_text = apps.get_model("custom_models", "ItemText")
+    # get db alias
+    db_alias = schema_editor.connection.alias
+    # get language objects
+    l_english = (
+        language.objects.using(db_alias)
+        .filter(ISO_Code_1="en", ISO_Code_2="en-ca")
+        .last()
+    )
+    l_french = (
+        language.objects.using(db_alias)
+        .filter(ISO_Code_1="fr", ISO_Code_2="fr-ca")
+        .last()
+    )
+    # get item_type objects
+    it_tree_view = (
+        item_type.objects.using(db_alias).filter(type_desc="tree-view").last()
+    )
+    it_tree_child = (
+        item_type.objects.using(db_alias).filter(type_desc="tree-child").last()
+    )
+
+    # get item objects
+    background_id = (
```

Comment:
same comment as above.

Reply:
See commit #https://github.com/code-for-canada/project-thundercat/pull/409/commits/1aaa156edc0ce576d6dee0418f832b3f6436486a

Reply:
fixed (saw it offline)